### DISCUSSION TITLE
ui/nav: Hamburger menu [feature request]

### DISCUSSION THREAD

**Discussion by seanmrnda** [adityatelange/hugo-PaperMod#943] [Ideas]
It is time to add a hamburger menu as an option on the navigation bar which will trigger during mobile view.

config.yml
```yml
hamBurgerMenu: true
```

Similar to your blog website.

![Screenshot_2022-07-14-08-25-17-27.png](https://user-images.githubusercontent.com/45195351/178859191-bf52a63f-abb3-49bd-9da8-af872856b7f1.png)

that's just a template in `layout/partials/nav.html` right?

**Comment by icy-comet** [comment]
@adityatelange has now shared the necessary snippets and steps to add the menu in the Discord channel. @Astrodevil @schwrzstrbn

**Reply by jesusfj710** [reply]
Can we get this in github too please?

**Reply by jesse-wei** [reply]
I copied Aditya's code from Discord into my repo. See [this commit](https://github.com/jesse-wei/jessewei.dev-PaperMod/commit/9d74c542e64699557cda562edacacccc700cd3c8).

**Reply by jesusfj710** [reply]
@jesse-wei I can't see the commit anymore, could you please re-share a Gist or something?

**Reply by jesse-wei** [reply]
> @jesse-wei I can't see the commit anymore, could you please re-share a Gist or something?

Sorry, I had changed the name of my repo. The link works now.

**Comment by jesusfj710** [comment]
qq, how would this works in a page were you have the menu and the languages?

For example, this:
![image](https://github.com/adityatelange/hugo-PaperMod/assets/71794881/c7abf310-f427-498d-a5b1-0231a0881ece)

I guess you will get the hamburger menu for the "sections", like blog, search, contact, home, etc only, right?

**Comment by Simon99de** [comment]
#1878
 I made a change to the hamburger menu. Unfortunately, I can't submit a pull request, and I'm also not sure whether the width should be specified manually when it switches, or whether it should be determined automatically by querying the actual width.
I didn't implement the language feature, but you could do the same thing there with a globe icon/SVG, or simply list the languages to the left of the menu.

I don’t need the language selection feature, since my website is only in one language.
